src/_common/user/timeout/timeout.model.ts

```diff
@@ -0,0 +1,12 @@
+import { Model } from '../../model/model.service';
+
+export class UserTimeout extends Model {
+	expires_on!: number;
+	reason!: string;
+
+	get isExpired() {
+		return this.expires_on <= Date.now();
+	}
```

Comment:
Is this actually reactive? I think the way to do this is to have an actual Node timer update a property and have the getter use that.